### PR TITLE
Improve VM search and logging

### PR DESCRIPTION
**Pull request by acernek**
Fixes issue #5

### CODE REVIEW

File: ovirtbmc/ovirtbmc.py [hunk 1]
```diff
@@ -43,29 +44,33 @@ def __init__(self, authdata, port, address, vm_name, cache_status, engine_fqdn,
         )
         self.vms_service = self.ovirt_conn.system_service().vms_service()
         self.vm = None
```
Comment (tinez) [inline ovirtbmc/ovirtbmc.py]: I think it would make more sense to have `self.vm_id` instead of `self.vm` now

File: ovirtbmc/ovirtbmc.py [hunk 2]
```diff
@@ -43,29 +44,33 @@ def __init__(self, authdata, port, address, vm_name, cache_status, engine_fqdn,
...
         # trying to find the vm indefinitely, since there's no point in continuing if we don't have an vm.
         while True:
             try:
-                self.vm = self._find_vm(vm_name)
-                if self.vm is not None:
-                    self.log(f'Managing vm: {vm_name} ID: {self.vm}')
-                    break
+                self.vm = self._find_vm(vm)
+                self.vm_name = self.vms_service.service(self.vm).get().name
+                self.log(f'Managing vm: {self.vm_name} ID: {self.vm}')
```
Comment (tinez) [inline ovirtbmc/ovirtbmc.py]: This log message doesn't align with the convention used below where we print out `f'{self.vm_name} ({self.vm})'`. Let's keep things consistent

File: ovirtbmc/ovirtbmc.py [hunk 3]
```diff
@@ -43,29 +44,33 @@ def __init__(self, authdata, port, address, vm_name, cache_status, engine_fqdn,
...
-    def _find_vm(self, vm_name):
-        vms = (vm for vm in self.vms_service.list() if vm.name == vm_name)
+    def _find_vm(self, vm):
         try:
-            vm_id = next(vms).id
-            return vm_id
-        except StopIteration:
-            self.log(f'Could not find specified vm {vm_name}')
-            sys.exit(1)
+            self.vms_service.service(vm).get()
```
Comment (tinez) [inline ovirtbmc/ovirtbmc.py]: Doesn't this return a `VmBase` instance? In line 70 we're returning the `id` directly

**Review by tinez — CHANGES_REQUESTED** [review]

File: ovirtbmc/ovirtbmc.py (hunk 1 above)
Comment (acernek) [reply ovirtbmc/ovirtbmc.py]: Done

File: ovirtbmc/ovirtbmc.py (hunk 2 above)
Comment (acernek) [reply ovirtbmc/ovirtbmc.py]: Done

File: ovirtbmc/ovirtbmc.py (hunk 3 above)
Comment (acernek) [reply ovirtbmc/ovirtbmc.py]: If this code works, then `vm` should be id and we can return it directly. Adjusted the code to make it clearer

**Review by michalskrivanek — APPROVED** [review]

**Pull request merged**